resources/modernizer_exclusions.txt

```diff
@@ -20,3 +20,4 @@ com/google/common/primitives/Longs.compare:(JJ)I
 com/google/inject/Inject
 com/google/inject/Provider
 java/lang/StringBuffer."<init>":()V
+java/util/Hashtable."<init>":(I)V
```

Comment:
No, I meant `HashMap`. The modernizer plugin forbids the use of Hashtable in favor of HashMap, but HashMap does not extend Dictionary. Thus it can't be used as an argument to the BundleContext.registerService method. That's why I just excluded it from modernizer, as we now actually need a Hashtable instance. See [this comment](https://github.com/jclouds/jclouds/pull/1033#issuecomment-258078992) for further details.